src/index.html

```diff
@@ -11,7 +11,182 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      class="form"
+      method="post"
+    >
+      <fieldset class="form__section">
+        <legend>Personal information</legend>
+        <div class="form__item">
+          <label>
+            Surname:
+            <input
+              type="text"
+              id="userSurname"
+              name="userSurname"
+            >
+          </label>
+        </div>
+        <div class="form__item">
+          <label>
+            Name:
+            <input
+              type="text"
+              id="userName"
+              name="userName"
+            >
+          </label>
+        </div>
+        <div class="form__item">
+          <label>
+            How old are You?
+            <input
+              type="number"
+              name="userAge"
+              id="userAge"
+              value="12"
+              max="100"
+              min="1"
+            >
+          </label>
+        </div>
+        <div class="form__item">
+          <label>
+            Full date of birth:
+            <input
+              type="date"
+              id="userBirth"
+              name="userBirth"
+            >
+          </label>
+        </div>
+        <div class="form__item">
+          <label>
+            I accept the term of the agreement
+            <input
+              type="checkbox"
+              id="userAgreement"
+              name="userAgreement"
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="form__section">
+        <legend>Registration</legend>
+        <div class="form__item">
+          <label>
+            E-mail:
+            <input
+              type="email"
+              id="userEmail"
+              name="userEmail"
+              placeholder="email@example.com"
+              required
+            >
+          </label>
+        </div>
+        <div class="form__item">
+          <label>
+            Password:
+            <input
+              type="password"
+              id="userPassword"
+              name="userPassword"
+              required
+              minlength="6"
+              maxlength="24"
+              autocomplete="off"
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="form__section">
+        <legend>An interesting fact about you!</legend>
+        <div class="form__item">
+          <label for="cats">Do you love cats?</label>
+          <input
+            type="radio"
+            name="cats"
+            id="catsYes"
+          >
+          <label for="catsYes">Yes</label>
+          <input
+            type="radio"
+            name="cats"
+            id="catsNo"
+          >
+          <label for="catsNo">No</label>
```

Comment:
labels still don't work🥲
```suggestion
          <label>
            Yes
            <input
              type="radio"
              name="cats"
            >
          </label>
          
          <label>
            No
            <input
              type="radio"
              name="cats"
            >
          </label>
```